flow_control/tests/test_flowcontrol.py

```diff
@@ -185,3 +185,58 @@ def test_author_view(self, fragment_mock):
         fragment_mock.assert_called()
         fragment_instance.add_javascript.assert_called()
         fragment_instance.initialize_js.assert_called()
+
```

Comment:
@juancamilom do you think this assertions correctly show what should be happening?

Reply:
@felipemontoya I've added 3 inline comments